piccolo/table.py

```diff
@@ -407,6 +416,52 @@ def get_related(self, foreign_key: t.Union[ForeignKey, str]) -> Objects:
             .first()
         )
 
+    # TODO - I might merge this with ``get_related``.
+    def get_m2m(self, m2m: M2M) -> M2MGetRelated:
+        """
+        Get all matching rows via the join table.
+
+        .. code-block:: python
+
+            >>> band = await Band.objects().get(name="Pythonistas")
+            >>> await band.get_m2m(Band.genres)
+            [<Genre: 1>, <Genre: 2>]
```

Comment:
Minor typo here it should be `await Band.objects().get(Band.name == "Pythonistas")` and for `line 428` I get result like this
`[<Genre: "id" SERIAL PRIMARY KEY NOT NULL>, <Genre: "id" SERIAL PRIMARY KEY NOT NULL>]`

Reply:
Good catch. I've fixed the typo.

The response should actually be `[<Genre: 1>, <Genre: 2>]` - this is a bug. I'll fix it.

Reply:
Great. When you changed `__repr__` method in `table.py` I get correct result `[<Genre: 1>, <Genre: 2>]`

Reply:
Cool, thanks